specification/repository.md

```diff
@@ -54,11 +54,25 @@ approval is granted, GDI repositories MUST NOT cut a GA release.
 - MUST lock the versions of all build dependencies (e.g. libraries, binaries,
   scripts, docker images) or vendor them; **EXCEPTION:** tools that are
   available out-of-the-box on the CI runner
+- To help keep dependencies up to date, the repo MUST be configured with
+[Dependabot](https://github.com/dependabot/dependabot-core) or [Renovate](https://github.com/apps/renovate).
+
+#### Dependabot
+
 - MUST enable [Dependabot alerts](https://docs.github.com/en/code-security/dependabot/dependabot-alerts/about-dependabot-alerts)
   - MUST grant access to alerts for the approvers and maintainers teams
   - MUST enable [Dependabot security updates](https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/about-dependabot-security-updates)
 - MUST configure [Dependabot version updates](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates)
 
+#### Renovate
```

Comment:
Would it make sense to add `renovatebot` to https://github.com/signalfx/gdi-specification/blob/3e793c167b8ccd42bbdd4167bb628ff1f9c646c9/.github/workflows/cla.yml#L30?